LCM/scripts/RemoveModule.py

```diff
@@ -74,17 +73,8 @@ def RemoveMirroredDirectory(moduleDir, omiDir):
     if os.path.isfile(state_conf_location):
         state_conf = ConfigParser.ConfigParser()
         try:
-            # try to read the PID of the worker and make a best effort attempt to kill it
-            # don't fail uninstall if it cannot be killed
-            state_conf.read(state_conf_location)
-            worker_process_id = state_conf.get(config_state_section, config_pid_key)
-            workspace_id = state_conf.get(config_state_section, workspace_id_key)
-            os.kill(int(worker_process_id), signal.SIGTERM)
-            # Typicallly the above kill should be sufficient becasue we should never reach as state where there are more
-            # than 1 workers. Since we introduced a new user to exclusively for worker, we can safely kill all processes
-            # that user which were started with the workspace id for additional safety
-            # Note: using "--full" option instead of "-f" may cause compatibility issue with older shells
-            subprocess.call(["sudo", "pkill", "-u", automation_user, "-f", workspace_id])
+            # Kill all processes running as nxautomation
+            subprocess.call(["sudo", "pkill", "-u", automation_user, ".*"])
```

Comment:
from a regex point of view, ".*" matches everything, while "" matches nothing. The former made more logical sense.